docs/learn/digital-assets/metadata-management/cascade-and-sense-support.md

```diff
@@ -46,14 +44,14 @@ if (result) {
 }
 
 // upload folder
-const results = await provider.uploadFolderToCascade("./examples");
+const results = await provider.uploadFolderToCascade('./examples');
```

Comment:
This has not been merged on data-providers yet. Checking it out. The commit needs some work.